components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -582,6 +582,14 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView
   }
 }
 
+- (CGFloat)topSafeArea {
+  if (@available(iOS 11.0, *)) {
+    return self.view.safeAreaInsets.top;
+  } else {
+    return kFixedStatusBarHeightOnPreiPhoneXDevices;
```

Comment:
_Questions:_ Looking at the way it's currently done, it looks like it accounts for landscape orientation having no status bar. Would we lose that capability?

Additionally, once MDCDeviceTopSafeAreaInset() is gone, would this constant value have to be defined in each component that requires it?

Reply:
1. On landscape the status bar is there and is also 20dp tall on non-iPhoneX devices.
2. Yes, I believe all components that were using MDCDeviceTopSafeAreaInset would have to define a similar constant.